liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java

```diff
@@ -273,18 +313,4 @@ protected SqlStatement getConnectionSchemaNameCallStatement() {
         return new RawCallStatement("select current_schema()");
     }
 
-    private boolean catalogExists(String catalogName) throws DatabaseException {
```

Comment:
Why did you take these methods out?

Reply:
Because those method are unused.

Reply:
Not necessarily. They may be used in appdba, stored logic, or some 3rd party extension. We can't assume a method isn't used in the liquibase code just because it's not used inside the main codebase.

Reply:
Ok Good catch! I'll revert those methods

Reply:
Good point, hadn't noticed they were private. That should be fine, then.